net.mkiol.SpeechNote.yaml

```diff
@@ -0,0 +1,428 @@
+        - -DTARGET=ARMV8
+      sources:
+        - type: archive
+          url: https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21.tar.gz
+          md5: ffb6120e2309a2280471716301824805
+
+    - name: whispercpp
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DBUILD_SHARED_LIBS=OFF
+        - -DWHISPER_BUILD_TESTS=OFF
+        - -DWHISPER_BUILD_EXAMPLES=OFF
+        - -DWHISPER_SUPPORT_OPENBLAS=ON
+        - -DCMAKE_C_FLAGS='-O3 -ffast-math -I$$C_INCLUDE_PATH/openblas'
+        - -DCMAKE_CXX_FLAGS='-O3 -ffast-math -I$$CPLUS_INCLUDE_PATH/openblas'
+      sources:
+        - type: archive
+          url: https://github.com/ggerganov/whisper.cpp/archive/ea1f8a50d4f70b54d9dd03205207a80019e243f9.zip
+          md5: 8bf8086d3c43966f8adc8e2d1596de42
+        - type: patch
+          path: patches/whispercpp.patch
+
+    - name: webrtcvad
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      sources:
+        - type: archive
+          url: https://github.com/webrtc-mirror/webrtc/archive/ac87c8df2780cb12c74942ec8a473718c76cb5b7.zip
+          md5: f1489c137b354594632d260978b283a4
+        - type: patch
+          path: patches/webrtcvad.patch
+
+    - name: xz
+      buildsystem: autotools
+      config-opts:
+        - --enable-xz
+        - --disable-xzdec
+        - --disable-lzmadec
+        - --disable-lzmainfo
+        - --disable-lzma-links
+        - --disable-scripts
+        - --disable-doc
+        - --disable-shared
+        - --enable-static
+        - --with-pic
+      sources:
+        - type: archive
+          url: https://downloads.sourceforge.net/lzmautils/xz-5.4.2.tar.gz
+          md5: 4ac4e5da95aa8604a81e32079cb00d42
+
+    - name: mbrola
+      buildsystem: simple
+      build-commands:
+        - make
+        - 'cp Bin/mbrola /app/bin'
+      sources:
+        - type: archive
+          url: https://github.com/numediart/MBROLA/archive/refs/tags/3.3.tar.gz
+          md5: 06993903c7b8d3a8d21cc66cd5a28219
+
+    - name: espeak
+      buildsystem: simple
+      build-commands:
+        - 'mkdir espeak'
+        - 'tar -x -C espeak --strip=1 -f lib/espeak-ng-1.52-patched.tar.gz'
+        - 'cd espeak && ./autogen.sh'
+        - 'cd espeak && ./configure --prefix=/app --with-pic --with-pcaudiolib=no --enable-static --disable-shared --disable-rpath --with-extdict-ru --with-extdict-zh --with-extdict-zhy'
+        - 'cd espeak && make install'
+      sources:
+        - type: archive
+          url: https://github.com/rhasspy/piper/archive/e967bdc1a838023fe3644f8ed7cf6e1e958b42ef.zip
+          md5: c9924cdb6305f6982171d486d5d633e1
+
+    - name: boost
+      buildsystem: simple
+      build-commands:
+        - './bootstrap.sh --prefix=/app --with-libraries=system'
+        - './b2 -j `nproc` install'
```

Comment:
don't use nproc. Use `$FLATPAK_BUILDER_N_JOBS`

Reply:
Fixed in 2be1a3a